anomalib/models/dfm/torch_model.py

```diff
@@ -129,11 +137,19 @@ def score(self, features: Tensor) -> Tensor:
             score = self.gaussian_model.score_samples(feats_projected)
         elif self.score_type == "fre":
             feats_reconstructed = self.pca_model.inverse_transform(feats_projected)
+            fre = torch.square(features - feats_reconstructed).reshape(feature_shapes)
+            fre_map = torch.unsqueeze(torch.sum(fre, dim=1), 1)
+            score_map = F.interpolate(fre_map, size=self.input_size, mode="bilinear", align_corners=False)
             score = torch.sum(torch.square(features - feats_reconstructed), dim=1)
         else:
             raise ValueError(f"unsupported score type: {self.score_type}")
 
-        return score
+        if self.score_type == "nll":
+            output = score
+        else:
+            output = score, score_map
```

Comment:
The torch inferencer expects the predictions of the torch model for the segmentation task type to be in the form (anomaly_map, anomaly_score), so these two should be reversed here (of course, the lightning model should then also be updated accordingly. 

This is what's causing the failing test case in the CI.